build: increase max ops for stale feature action

mhdawson opened the pull request:
Set the limit to match the number used in the
close stalled action. Required as not all eligeable
stale features are being processed.

Signed-off-by: Michael Dawson <mdawson@devrus.com>

<!--
Before submitting a pull request, please read
https://github.com/nodejs/node/blob/HEAD/CONTRIBUTING.md.

Commit message formatting guidelines:
https://github.com/nodejs/node/blob/HEAD/doc/contributing/pull-requests.md#commit-message-guidelines

For code changes:
1. Include tests for any bug fixes or new features.
2. Update documentation if relevant.
3. Ensure that `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
-->

mhdawson commented:
Due to the ways the days stale is set, this will not affect how much of the backlog is processed. We agreed on doing 20 twice/week and I'll continue to manager that by PRs to update the stale day to achieve that.

targos commented:
Could we add a parameter to `workflow_dispatch` so it's possible to run with different number of days without opening PRs?

mhdawson commented:
@targos I'm not an expert on actions. We have workflow_dispatch which was added so that we could run manually which I've used for test. I think what you are saying is that we can add a parameter to that ?

Mesteery commented:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs

So we could have something like this for example:
```yaml
    inputs:
      operations-per-run:
        description: Operations per run    
        required: false
        default: 500
        type: number
      days-before-stale:
        description: Idle number of days before marking feature requests stale
        required: false 
        default: 906
        type: number
```

mhdawson commented:
@Mesteery thanks. I think we could limit to just the part for the days before stale. I've updated the PR to add the parameter and removed the cron.  I don't know how to pass the parameter for the cron and we could have to change once we are caught up anyway.

This should mean ones this lands we only need one more PR to add the cron back and then we should be done.

@targos

Mesteery commented on .github/workflows/close-stale-feature-requests.yml:
```diff
@@ -1,9 +1,11 @@
-  schedule:
-    # Run every day at 1:00 AM UTC.
-    - cron: 0 1 * * *
+    daysBeforeStale:
+    description: Idle number of days before marking feature requests stale
+    required: true 
+    default: 906
+    type: number
```
```suggestion
    inputs:
      daysBeforeStale:
        description: Idle number of days before marking feature requests stale
        required: true 
        default: 906
        type: number
```

VoltrexMaster commented on .github/workflows/close-stale-feature-requests.yml:
```diff
@@ -1,9 +1,12 @@
+    inputs:
+      daysBeforeStale:
+        description: Idle number of days before marking feature requests stale
+        required: true 
```
```suggestion
        required: true
```
Lint.

mhdawson commented:
@VoltrexMaster thanks for the whitespace fix

mhdawson commented:
Landed in a62972ab287d
